packages/appium/lib/schema/cli-transformers.js

```diff
@@ -85,20 +90,18 @@ export const transformers = {
   json: (jsonOrPath) => {
     let json = jsonOrPath;
     let loadedFromFile = false;
-    try {
-      // use synchronous file access, as `argparse` provides no way of either
-      // awaiting or using callbacks. This step happens in startup, in what is
-      // effectively command-line code, so nothing is blocked in terms of
-      // sessions, so holding up the event loop does not incur the usual
-      // drawbacks.
-      json = readFileSync(jsonOrPath, 'utf8');
-      loadedFromFile = true;
-    } catch (err) {
-      // unreadable files don't count.
-      // also `ENAMETOOLONG` can happen if we try to open a file that's a huge JSON string.
-      if (err.code !== 'ENOENT' && err.code !== 'ENAMETOOLONG') {
-        throw err;
+    if (existsSync(jsonOrPath)) {
```

Comment:
will existsSync throw if the path is super long? or will this get around the nametoolong error?

Reply:
I don't thinks Node.js devs would allow such issue. But lets try:

```
Welcome to Node.js v18.6.0.
Type ".help" for more information.
> const fs = require('fs');
undefined
> fs.existsSync('x'.repeat(10000))
false
> 
```